### PR TITLE
Added support for string data type.

### PR DESCRIPTION
**Pull request by DhobaleVishal**
Signed-off-by: Vishal Dhobale <vishal.dhobale@seagate.com>

# Problem Statement
-   Problem statement

# Design
-   For Bug, Describe the fix here.
-   For new test addition post the link for design

# Coding
   Checklist for Author
-   [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-   [x] New/Affected tests are executed
-   [x] Attach test execution logs

# Review 
  Checklist for Author
-   [x] JIRA number/GitHub Issue added to PR
-   [x] PR is self reviewed
-   [x] Jira and state/status is updated and JIRA is updated with PR link
-   [x] Check if the description is clear and explained

# Documentation 
  Checklist for Author
-   [ ] Changes done to ReadMe

### CODE REVIEW

File: src/commons/utils/corio_utils.py [hunk 1]
```diff
@@ -125,21 +128,34 @@ def run_local_cmd(cmd: str) -> tuple:
...
-    :param size: Size in bytes
-    :param file_name: File name
+    :param size: Size in bytes.
+    :param file_name: File name or file path.
+    :param data_type: supported data type string(str)/byte(bytes) while create file.
     """
     base = KIB * KIB
-    file_path = os.path.join(DATA_DIR_PATH, file_name)
+    if os.path.isdir(os.path.split(file_name)[0]):
+        file_path = file_name
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py:142]: Please explain the need of this if statement.

Comment (DhobaleVishal) [reply src/commons/utils/corio_utils.py:142]: if absolute file path passed then to handle such case this condition introduced.

File: src/commons/utils/corio_utils.py [hunk 2]
```diff
@@ -125,21 +128,34 @@ def run_local_cmd(cmd: str) -> tuple:
         return False, error
 
 
-def create_file(file_name: str, size: int) -> str:
+def create_file(file_name: str, size: int, data_type: str or bytes = bytes) -> str:
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py]:
```suggestion
def create_file(file_name: str, size: int, data_type: Union[Type[str], Type[bytes]] = bytes) -> str:
```

File: src/commons/utils/corio_utils.py [hunk 3]
```diff
@@ -125,21 +128,34 @@ def run_local_cmd(cmd: str) -> tuple:
...
+        file_path = os.path.join(DATA_DIR_PATH, file_name)
     while size > base:
-        with open(file_path, 'ab+') as f_out:
-            f_out.write(os.urandom(base))
+        if issubclass(data_type, bytes):
+            with open(file_path, 'ab+') as bf_out:
+                bf_out.write(os.urandom(base))
+        else:
+            with open(file_path, 'a+', encoding="utf-8") as sf_out:
+                sf_out.write(b64encode(os.urandom(base)).decode("utf-8"))
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py]: This does not generate text file of the given size. Please verify.

**Review by ketanarlulkar — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**